coderd/database/dbauthz/dbauthz.go

```diff
@@ -1130,6 +1184,13 @@ func (q *querier) CleanTailnetTunnels(ctx context.Context) error {
 	return q.db.CleanTailnetTunnels(ctx)
 }
 
+func (q *querier) CountInProgressPrebuilds(ctx context.Context) ([]database.CountInProgressPrebuildsRow, error) {
+	if err := q.authorizeContext(ctx, policy.ActionRead, rbac.ResourceSystem); err != nil {
```

Comment:
`ResourceSystem` should not be used for new things. It's a random assortment of stuff and we should not add to the mess. One of the goals of access control is to give actors the least least privilege they need to do their work, and we can't do that effectively if we are lumping low-security stuff like counts and metrics in with extremely high-security stuff like crypto keys.

Maybe a new `ResourcePrebuildsStatistics` for `CountInProgressPrebuilds`, `GetPrebuildMetrics` and `GetPresetsBackoff`.

Reply:
@spikecurtis we solved this using existing resources

Reply:
@spikecurtis @Emyrk Fixed: https://github.com/coder/coder/pull/16891/commits/ea9c53b1a0b64f6724aaff87416065a31bc076a8